data_collection/sns_scrape.py

```diff
@@ -0,0 +1,33 @@
+# We are going to scrape Twitter using snscrape
```

Comment:
To make sure the lint is passing, the python file line 1 needs to be a docstring to explain what the .py file is about.
```suggestion
"""We are going to scrape Twitter using snscrape"""
```